### PR TITLE
HDS-1628 Zooming makes koro svg lines appear bug

### PR DESCRIPTION
**Pull request by laitine**
## Description

* When the koro SVG shape is rotated or vertically flipped random thin border lines can appear, this fixes such issues for Chrome, Firefox and Safari
* The SVG shapes have been updated to accurately reflect the UI kit and SVG files optimized to improve performance
* Core and React implementation for koros including Storybook stories

## Motivation and Context

This fixes the issue with visual integrity of the koro shape when the rotated or vertically flipped and improves consistency by accurately replicating the koro graphic asset designs defined by the brand. 

## How Has This Been Tested?

Manual testing on latest versions of Chrome, Firefox and Safari on a Mac. Snapshot tests and visual tests.

## Screenshots (if appropriate):

Koro issue screenshot

### CODE REVIEW

**Review by mikkojamG — CHANGES_REQUESTED** [review]
Looks good 👍 Unit tests need snapshots to be updated, possibly also visual tests 🤔

File: packages/react/src/components/koros/Koros.tsx [hunk 1]
```diff
@@ -39,19 +39,19 @@ export type KorosProps = {
...
+        d="m0 21v64h32v-64c-4 0-5.4-4-5.4-4l-5.2-13s-1.4-4-5.4-4-5.4 4-5.4 4l-5.2 13s-1.4 4-5.4 4z"
       />
     ),
-    pulse: <path transform={transform} d="M0,800h20V0c-5.1,0-5.1,6.4-10,6.4S4.9,0,0,0V800z" />,
-    storm: <path transform={transform} d="M20,800V0c-2.3,5.5-8.7,8.1-14.3,5.7C3.1,4.7,1.2,2.6,0,0v800H20z" />,
-    wave: <polygon transform={transform} points="0,800 20,800 20,0 9.8,10.1 0,0 " />,
-    calm: <path transform={transform} d="M788 0.279785H0V109.739H788V0.279785Z" />,
+    pulse: <path transform={transform} d="m0 10v75h32v-75c-8 0-8-10-16-10s-8 10-16 10z" />,
+    storm: <path transform={transform} d="m0 10v75h32v-75c-8 0-13-3.7-16-10-3 6.3-8 10-16 10z" />,
+    wave: <path transform={transform} d="m0 0v85h32v-85l-16 16z" />,
```
[the remark this answers is not in the document]
Comment (laitine) [reply packages/react/src/components/koros/Koros.tsx:53]: These were aligned to before 3.0 but I can update them to the new naming scheme 👌

**Comment by laitine** [comment]
Pending official term translations from brand to use for names https://helsinkisolutionoffice.atlassian.net/browse/HDS-1505

File: packages/react/src/components/koros/Koros.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (harriplappalainen) [reply packages/react/src/components/koros/Koros.tsx:53]: Fixes need to be added to the branch `release-3.0.0`. Also Wave should be Vibration and Storm should be Wave.

**Comment by laitine** [comment]
Core components also with correct names updated. Snapshot test and visual tests now added.

**Review by harriplappalainen — APPROVED** [review]
Is good! 👍 🥇 Except for the conflicts 😞

**Comment by laitine** [comment]
Missing implementation for Koros Exact fit in React https://hds.hel.fi/storybook/react/?path=/docs/components-koros--exact-fit

**Review by mikkojamG — APPROVED** [review]

**Pull request merged**